ibind/base/rest_client.py

```diff
@@ -159,7 +212,9 @@ def request(self, method: str, endpoint: str, attempt: int = 0, log: bool = True
         # we repeat the request attempts in case of ReadTimeouts up to max_retries
         for attempt in range(self._max_retries + 1):
             try:
-                response = requests.request(method, url, verify=self.cacert, timeout=self._timeout, **kwargs)
+                # add IBKR OAuth headers to request function
+                response = requests.request(method, url, headers=headers, params=params, timeout=10, **kwargs,verify=False)
+                # response = requests.request(method, url, verify=self.cacert,headers=header_oauth,params=params, timeout=self._timeout, **kwargs)
```

Comment:
Probably now is a good time to restore all functionalities in this call. So add `timeout=self._timeout` and `verify=self.cacert`.

Reply:
I agree